project/epi/models.py

```diff
@@ -867,18 +867,18 @@ def flat_complete_header_row():
             "exposure-sampling_period",
             "exposure-age_of_exposure",
             "exposure-duration",
-            "exposure-exposure_distribution",
             "exposure-n",
             "exposure-estimate",
+            "exposure-estimate_type",
             "exposure-variance",
+            "exposure-variance_type",
             "exposure-lower_ci",
             "exposure-upper_ci",
             "exposure-lower_range",
             "exposure-upper_range",
             "exposure-lower_bound_interval",
             "exposure-upper_bound_interval",
-            "exposure-estimate_type",
-            "exposure-variance_type",
+            "exposure-exposure_distribution",
```

Comment:
@JoshAddington had to move `exposure-exposure_distribution` column as the current implementation produced an excel file where data and headers were unsynced.